src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+  const stylesObject = {};
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
+
+    if (property && value) {
```

Comment:
The check for both `property` and `value` being truthy is good, as it prevents adding invalid entries to the `stylesObject`. Keep in mind that this will exclude valid CSS properties with a value of `0`, which is falsy in JavaScript. You may want to explicitly check for an empty string instead.